src/browser/monkey-menu.js

```diff
@@ -158,6 +158,29 @@ async function onHashChange(event) {
 }
 
 
+function onClick(event) {
+  if (event.which === 3) {  // Right mouse click.
+    if (!gActiveUuid) {
+      const elements = document.querySelectorAll(':hover');
+      // Last element is the name of the script; last but one is the anchor.
+      const hash = elements[elements.length - 2].hash;
```

Comment:
Is query selector deterministic? It might be safer to go up from `event.target` looking for `a` tags. Perhaps [`el.closest()`](https://developer.mozilla.org/en-US/docs/Web/API/Element/closest). Or a loop like:
```javascript
while ((el = el.parentNode) && el.tagName !== 'A')
// Will have null or the element. Bail if null.
```

Reply:
Slight problem with my while loop. Flip the parameters around and use `.parentElement` instead.
```javascript
while (el.tagName !== 'A' && (el = el.parentElement))
```

Reply:
I didn't know how to obtain the anchor, so I googled a bit and tested several solutions given. Then I found this solution (for not-so-old browsers). It returns an array (rather a stack) of all elements from `<html>` over `<body>` to `<a>` and `#text`. Almost exactly what I wanted. IIRC `event.target` (which would be a perfect solution) didn't work. I should test that again...

Reply:
Oops, strange. Now it works! This is much easier:
`event.target.parentElement`

Reply:
> `event.target.parentElement`

I don't think this is guaranteed to be the correct element. If I remember correctly clicks are first sent to the lowest node in the tree, that you're hovering over (which is the target). If you click over the icon then your parent node is going to be `div.icon`. The safest, and more robust in case of changes to the HTML, would be to do some sort of parental search.

Reply:
I wanted the area with the script name as target, so its parent is always the anchor.

Reply:
Hmm, okay that works fine with the icons. But now I need a check against null...